### PR TITLE
fix: fix token test

### PR DESCRIPTION
**Pull request by Bento007**
### Reviewers
**Functional:** @danieljhegeman 

**Readability:** 

---

## Changes
- Cleaning up the openapi specs
- Clean up collection handlers
- fix token generation test
- add tests case where API key is valid but no longer stored in Auth0

### CODE REVIEW

File: backend/config/curation-api.yml [hunk 1]
```diff
@@ -43,8 +43,7 @@ paths:
       summary: Fetch Collections metadata.
       description: Fetch all Collections metadata. Authorization required for private collections.
       security:
-        - cxguserCookie: []
-        - curatorAccess: []
+        - curatorAccessLenient: []
```
Comment (danieljhegeman) [inline backend/config/curation-api.yml:46]: we need to let all requests through but we check with user before adding private collections to response, right

File: backend/config/curation-api.yml [hunk 2]
```diff
@@ -123,28 +121,11 @@ paths:
...
-        - $ref: "#/components/parameters/path_collection_uuid"
-      responses:
-        "202":
-          description: Accepted
-          content:
-            application/json:
-              schema:
-                type: string
-        "401":
-          $ref: "#/components/responses/401"
```
Comment (danieljhegeman) [inline backend/config/curation-api.yml:141]: and we'll add this back in later

File: backend/corpora/lambdas/api/v1/curation/auth/token.py [hunk 3]
```diff
@@ -16,5 +16,7 @@ def post():
         raise UnauthorizedError("The API key is invalid")
     else:
         identity = auth0_management_session.get_user_api_key_identity(token["sub"])
+        if not identity:
+            raise NotFoundHTTPException("The API key is has is no longer valid.")
```
Comment (danieljhegeman) [inline backend/corpora/lambdas/api/v1/curation/auth/token.py:20]: typo

**Review by danieljhegeman — APPROVED** [review]
see typo

File: backend/config/curation-api.yml (hunk 2 above)
Comment (Bento007) [reply backend/config/curation-api.yml:141]: when we decide to support it

**Pull request merged**